pkg/autodiscovery/autoconfig.go

```diff
@@ -541,6 +541,10 @@ func (ac *AutoConfig) resolveTemplateForService(tpl integration.Config, svc list
 
 // GetLoadedConfigs returns configs loaded
 func (ac *AutoConfig) GetLoadedConfigs() map[string]integration.Config {
+	if ac.store == nil {
+		log.Error("Autoconfig store not initialized")
```

Comment:
can the user see this message when they run `configcheck` before the Agent is initialized?

(IMO, the user should see a warning/error message in the command's output that tells them what's going on. If it's too complex to add let's not block the PR on this though)

Reply:
@olivielpeau totally agree that this should ideally be more explicit, I've looked into it this would require an mount of code change that I'm not comfortable with on a freeze day.
I've only seen this crash once in a `flare` in which we would have got the logs.